lineman/app/components/groups_page/groups_loaded.coffee

```diff
@@ -0,0 +1,15 @@
+angular.module('loomioApp').directive 'groupsLoaded', ($timeout) ->
+  restrict: 'A'
+  replace: true
+  link: (scope, element) ->
+    $timeout ->
+      leftHeight = rightHeight = 0
```

Comment:
Hey I really like this chunk of code. Yeah it's a little javascript, but it's super light and allows the 2-column group page design to be possible in a general case, by dropping groups into a column based on which column is shorter at the time.